<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Dell-Home-laptop" failures="1" tests="2" name="Loginedge" time="134.385" errors="0" timestamp="2024-11-28T11:50:10 AEST">
  <testcase classname="testCases.TC001_LogInSuccess" name="Veiry_Login_Success" time="76.368">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC001_LogInSuccess.Veiry_Login_Success(TC001_LogInSuccess.java:41)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1623)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- Veiry_Login_Success -->
  <testcase classname="testCases.TC002_LogInFail" name="Verify_Login_Fail" time="3.73"/>
</testsuite> <!-- Loginedge -->
